tests/phpunit/tests/query/cacheResults.php

```diff
@@ -1282,7 +1282,7 @@ public function data_author_cache_warmed_by_the_loop() {
 	/**
 	 * Ensure lazy loading term meta queries all term meta in a single query.
 	 *
-	 * @since 6.1.2
+	 * @since 6.2.0
```

Comment:
```suggestion
	 * @since 6.2.0
	 *
```

Add space between two tags.